### ISSUE TITLE
Add Star, Prime, and Prism Star subtype fields

### ISSUE THREAD

**Issue by bradley-erickson** [PokemonTCG/pokemon-tcg-data#283]
I believe the API would benefit from adding the Star, Prime, and Prism Star fields to the subtype category. These are significant side mechanics that I believe people would like to be able to search on. Adding them to the subtype category feels appropriate.
See attached images for which cards get what
Star cards:
![image](https://user-images.githubusercontent.com/26288699/164987221-56416c7b-5337-4b62-a68d-73381e714422.png)

Prime cards:
![image](https://user-images.githubusercontent.com/26288699/164987256-4d423f7d-8454-422a-9bb9-2e8f1fb89fd7.png)

Prism Star cards:
![image](https://user-images.githubusercontent.com/26288699/164987288-92f69fa4-57ec-41b2-872c-8e1babe807cb.png)

**Comment by bradley-erickson** [comment]
I also think it would be worth it to add an SP category.

**Comment by mmxxiii** [comment]
> I also think it would be worth it to add an SP category.

I had suggested that way back in issue #27 . But at that time, the subtype field didn't exist.

I'd also like to add the SP subtypes: Team Galactic's Pokémon (G), Frontier Brain's Pokémon (FB), Champion's Pokémon (C), Gym Leader's Pokémon (GL), and Elite Four's Pokémon (4).
There also Movie Pokemon (M), but those were released in Japanese only.

I'd be happy to take on this project if there are no objections.
